reporters/ClickhouseReporter.js

```diff
@@ -95,47 +95,55 @@ const TABLE_DEFINITIONS = [
     PRIMARY KEY(crawlId, pageId, targetId)`,
 ];
 
+/**
+ * @param {string | string[]} args
+ */
+function santizeCallArgs(args) {
+    // in some cases call args have been stringified, so unwrap that first.
+    const argsArray = typeof args === 'string'  ? JSON.parse(args) : (args || []);
+    return argsArray.map((/** @type {string} */ s) => s.replace(/'/g, ''));
+}
+
 class ClickhouseReporter extends BaseReporter {
 
     id() {
         return 'clickhouse';
     }
 
     /**
-     * @param {{verbose: boolean, startTime: Date, urls: number, logPath: string, regionCode?: string}} options 
+     * @param {{verbose: boolean, startTime: Date, urls: number, logPath: string}} options 
```

Comment:
we are still passing regionCode here from `crawl-cli` and it still shows up in BaseReporter - should we drop it from all the places?